### DISCUSSION TITLE
Capstone Telebot

### DISCUSSION THREAD

**Discussion by rastitiratnasari** [teamalgoritma/community#828] [Q&A] [answered]
Bagaimana cara mengatasi error dibawah ini 

----> [5](vscode-notebook-cell:?execution_count=77&line=5) start_date = df_campaign['reporting_date'].min().dt.strftime(format="%d %b %Y")
      [6](vscode-notebook-cell:?execution_count=77&line=6) end_date = df_campaign['reporting_date'].max().strftime(format="%d %b %Y")
      [8](vscode-notebook-cell:?execution_count=77&line=8) # DO NOT EDIT (FOR CHECKING PURPOSE)

AttributeError: 'NaTType' object has no attribute 'dt'

![Screenshot (15)](https://github.com/teamalgoritma/community/assets/150925524/7f7c125a-9a66-40cb-87e0-9e4249e9688d)

**Comment by awqigibran** [comment, marked as answer]
Saya sempat mengalami hal yang sama. Di case saya, ternyata saya ada kesalahan ketika meng-assign tipe data kolom campaign_id di DataFrame df. Setelah di-assign menjadi string (str) problem di atas bisa resolve. Hope it helps
